### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by Aiozlove** [winstxnhdw/lc-hax#151]
Do you have a compilation tutorial because I am a fool.

**Comment by winstxnhdw** [comment]
No, there isn't but don't worry. There's dozens of such cases if you look around `Discussions`.
